api.go

```diff
@@ -341,6 +342,11 @@ func (api *API) GetProducers() (out *ProducersResp, err error) {
 	return
 }
 
+func (api *API) GetAbiBinToJson(code AccountName, action ActionName, binargs string,) (out interface{}, err error) {
```

Comment:
Do you want to keep this part of this PR?

Reply:
I will add this in another PR.